.github/workflows/merge-release-notes.yaml

```diff
@@ -0,0 +1,61 @@
+name: Merge release notes
+
+on:
+  workflow_dispatch:
+
+jobs:
+  merge-release-notes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
+          echo $VERSION
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
```

Comment:
[trivial] I just noticed that these lines use both single quotes and double quotes. Could we use a consistent type of quotation mark for both lines?

Reply:
Ah, my bad. I fixed it at 9100a608.